rules/crowdstrike_rules/crowdstrike_real_time_response_session.py

```diff
@@ -1,29 +1,29 @@
-from panther_base_helpers import deep_get
+from panther_base_helpers import deep_get, get_crowdstrike_field
+
+MATCHES = ["Event_RemoteResponseSessionStartEvent"]
 
 
 def rule(event):
-    return (
-        deep_get(event, "unknown_payload", "ExternalApiType", default="<unknown-ExternalApiType>")
-        == "Event_RemoteResponseSessionStartEvent"
-    )
+    return get_crowdstrike_field(event, "ExternalApiType") in MATCHES
```

Comment:
Curious, why did you not perform an equality check? If `None` creates a problem you could solve this with `default=""` as parameter in the call to `get_crowdstrike_field` I guess?

Reply:
Fixed.

Reply:
Resolving, as it's old and has been done.